src/features/content-explorer/content-explorer/ContentExplorerNewFolderButton.js

```diff
@@ -16,9 +16,10 @@ const ContentExplorerNewFolderButton = ({
     isCreateNewFolderAllowed = true,
 }) => {
     const doesModeAllowCreateNewFolder =
-        contentExplorerMode === ContentExplorerModes.SELECT_FOLDER ||
+        contentExplorerMode === ContentExplorerModes.COPY ||
```

Comment:
Are we sure this is something we want to support for copy?

Reply:
I feel like it should be ok because there are other props which can control the showing and hiding of the create folder button as well, this just makes it an option for `copy` mode, if so desired